packages/components/src/components/hds/icon/types.ts

```diff
@@ -0,0 +1,30 @@
+export enum HdsIconSizeValues {
+  Sixteen = '16',
+  TwentyFour = '24',
+}
+
+export type HdsIconSizes = `${HdsIconSizeValues}`;
+
+export enum HdsIconColorValues {
```

Comment:
(question): wouldn't we want all of these to be HdsColorValues anyway? Is there a need to have them specifically in IconColorValues? It seems reasonable that if these are our ColorValues we'd want them available for everywhere that we use these values. WDYT?

Reply:
I thought about it, but I ended up keeping them separate for different reasons:

1) we may want to add also the "brand" colors for the icon (but not the text, I don't see use cases for that); at the moment thought is not clear _how_ we would do it: have [different brand colors](https://helios.hashicorp.design/foundations/tokens?searchQuery=brand), which ones would be the ones that _could_ be used for an icon?
2) this would require to add a global `types.ts` file, under `packages/components/src/components/hds`; not a big deal but maybe something we want to consider a bit more

so my thinking is that we can always do it, it's not something for this specific PR